src/common/tiles/BigTile/index.js

```diff
@@ -20,58 +22,103 @@ import {
   Article,
 } from "./styled";
 import noPicture from "../PersonTile/noPicture.png";
+import noPoster from "../MovieTile/noPoster.png";
 
 export const BigTile = ({
   id,
   poster_path,
   title,
   release_date,
+  place_of_birth,
+  birthday,
   rating,
   votes,
   countries,
   article,
+  genres,
+  profile_path,
 }) => {
   return (
     <MainContainer id={id}>
-      <Image
-        src={poster_path ? `${imageBaseUrl}/w342${poster_path}` : noPicture}
-        alt={`poster of ${title}`}
-      />
+      {poster_path && (
+        <Image
+          src={poster_path ? `${imageBaseUrl}/w342${poster_path}` : noPoster}
+          alt={`poster of ${title}`}
+        />
+      )}
+
+      {profile_path && (
+        <Image
+          src={profile_path ? `${imageBaseUrl}/w342${profile_path}` : noPicture}
+          alt={`portrait of ${title}`}
+        ></Image>
+      )}
```

Comment:
Ogólnie jest to dobrze zrobione ale wygląda dość dziwnie i myląco. Myślę, że na tą chwilę może zostać i potem postaramy się napisać to lepiej.  Zamknięcie img masz różne ;)